File-Formats/DocIO/word-to-markdown.md

```diff
@@ -0,0 +1,800 @@
+</tr>
+</tbody>
+</table>
+
+### Paragraph items
+
+<table style="width: 760px;">
+<thead>
+<tr style="height: 13px;">
+<td style="width: 262.889px; height: 13px;"><strong>Element in Word document</strong></td>
+<td style="width: 479.111px; height: 13px;"><strong>Notes</strong></td>
+</tr>
+</thead>
+<tbody>
+<tr style="height: 13px;">
+<td style="width: 262.889px; height: 13px;">Field</td>
+<td style="width: 479.111px; height: 13px;">Field result is preserved.</td>
+</tr>
+<tr style="height: 13px;">
+<td style="width: 262.889px; height: 13px;">Form Field</td>
+<td style="width: 479.111px; height: 13px;">Result of text and dropdown form fields are preserved.</td>
+</tr>
+<tr style="height: 13px;">
+<td style="width: 262.889px; height: 13px;">Hyperlink</td>
+<td style="width: 479.111px; height: 13px;">-</td>
+</tr>
+<tr style="height: 13px;">
+<td style="width: 262.889px; height: 13px;">Image</td>
+<td style="width: 479.111px; height: 13px;">-</td>
+</tr>
+<tr style="height: 13px;">
+<td style="width: 262.889px; height: 13px;">Horizontal rule</td>
+<td style="width: 479.111px; height: 13px;">-</td>
+</tr>
+<tr style="height: 13px;">
+<td style="width: 262.889px; height: 13px;">Content Control</td>
+<td style="width: 479.111px; height: 13px;">
+<p>Contents inside control controls are preserved as normal text.</p>
+<p>If first item of paragraph is checkbox content control, then it preserves as task item.</p>
+</td>
+</tr>
+<tr style="height: 13.6667px;">
+<td style="width: 262.889px; height: 13.6667px;">Breaks</td>
+<td style="width: 479.111px; height: 13.6667px;">Line and text wrapping breaks are supported.</td>
+</tr>
+<tr style="height: 13px;">
+<td style="width: 262.889px; height: 13px;">List</td>
+<td style="width: 479.111px; height: 13px;">To restart numbering for the continuous list in output markdown syntax, you should specify non-empty paragraph in between the two list in the Word document.</td>
+</tr>
+</tbody>
+</table>
+
+### Formatting
+
+<table style="width: 760px;">
+<thead>
+<tr>
+<td style="width: 318.625px;"><strong>Element in Word document</strong></td>
+<td style="width: 424.375px;"><strong>Notes</strong></td>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td style="width: 318.625px;">Bold</td>
+<td style="width: 424.375px;">-</td>
+</tr>
+<tr>
+<td style="width: 318.625px;">Italic</td>
+<td style="width: 424.375px;">-</td>
+</tr>
+<tr>
+<td style="width: 318.625px;">StrikeThrough</td>
+<td style="width: 424.375px;">-</td>
+</tr>
+<tr>
+<td style="width: 318.625px;">Subscript and Superscript</td>
+<td style="width: 424.375px;">-</td>
+</tr>
+<tr>
+<td style="width: 318.625px;">Hidden</td>
+<td style="width: 424.375px;">Considered as comments in Markdown syntax</td>
+</tr>
+<tr>
+<td style="width: 318.625px;">List</td>
+<td style="width: 424.375px;">Numbered and bulleted lists are supported.</td>
+</tr>
+</tbody>
```

Comment:
Add Limitations

Reply:
I maintained each limitation in each area of occurrence.